### PR TITLE
QuickViewer: Add version 1.1.6

### PR DESCRIPTION
**Pull request by linsui**
Close https://github.com/lukesampson/scoop-extras/issues/2755

Persist doesn't works, so copy files manully.

### CODE REVIEW

File: bucket/quickviewer.json [hunk 1]
```diff
@@ -0,0 +1,57 @@
+{
+    "version": "1.1.6",
+    "homepage": "https://kanryu.github.io/quickviewer/",
+    "description": "An image/comic viewer application.",
```
Comment (Ash258) [inline bucket/quickviewer.json]:
```suggestion
    "description": "An image/comic viewer.",
```

File: bucket/quickviewer.json [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+            "    if (!(Test-Path \"$persist_dir\\$file\")) {",
+            "        New-Item \"$persist_dir\\$file\" -Force | Out-Null",
+            "    }",
+            "    Copy-Item \"$persist_dir\\$file\" \"$dir\\$file\" -Force",
+            "}"
+        ]
+    },
+    "uninstaller": {
+        "script": [
+            "foreach ($file in @('progress.ini', 'quickviewer.ini')) {",
```
Comment (Ash258) [inline bucket/quickviewer.json]:
```suggestion
            "Copy-Item \"$dir\\*\" \"$persist_dir\" -Include 'progress.ini', 'quickviewer.ini'",
```

File: bucket/quickviewer.json [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+            "    if (!(Test-Path \"$persist_dir\\$file\")) {",
+            "        New-Item \"$persist_dir\\$file\" -Force | Out-Null",
+            "    }",
+            "    Copy-Item \"$persist_dir\\$file\" \"$dir\\$file\" -Force",
+            "}"
+        ]
+    },
+    "uninstaller": {
+        "script": [
```
Comment (Ash258) [inline bucket/quickviewer.json]: I forgot force, sorry..
```suggestion
        "script": "Copy-Item \"$dir\\*\" \"$persist_dir\" -Include 'progress.ini', 'quickviewer.ini' -Force"
```

Comment (linsui) [reply bucket/quickviewer.json]: How about `"Copy-Item \"$dir\\*.ini\" \"$persist_dir\" -Force"` ? And it works well without force.

Comment (Ash258) [reply bucket/quickviewer.json]: I do not know what ini files are there. If you want to persist only these two, then just copy these two.

Comment (linsui) [reply bucket/quickviewer.json]: There is no other ini file. We can simplify it as we do not need to specify which ini file should be persist.

Comment (Ash258) [reply bucket/quickviewer.json]: There is nothing which could be earned when copying all ini files. Yeah funciton will be shorter, but that is all.

Comment (linsui) [reply bucket/quickviewer.json]: I wonder will this make the installer more clear?

```
    "installer": {
        "script": [
            "if (!(Test-Path \"$persist_dir\")) { New-Item \"$persist_dir\" -ItemType Directory -Force | Out-Null }",
            "Copy-Item \"$persist_dir\\*.ini\" \"$dir\" -Force"
        ]
    },
    "uninstaller": {
        "script": "Copy-Item \"$dir\\*.ini\" \"$persist_dir\" -Force"
    },
```

Comment (Ash258) [reply bucket/quickviewer.json]: How will this solve that these configuration files do not exists in archive?

Comment (Ash258) [reply bucket/quickviewer.json]: Keep it as is now.

Comment (linsui) [reply bucket/quickviewer.json]: They will be created when first run. So we can persist them only when they exist. We can also do this on those file which can't be created manully. Such as

```
            "    if (Test-Path \"$persist_dir\\$file\") {",
            "        Copy-Item \"$persist_dir\\$file\" \"$dir\\$file\" -Force",
            "    }",
```

Comment (Ash258) [reply bucket/quickviewer.json]: These special use cases will be covered in lukesampson/scoop#3212

Until persistence is reworked keep it as simple and readable as you can.

Comment (linsui) [reply bucket/quickviewer.json]: Great! Thanks!

**Pull request merged**